time time

# Joint state
float64[] joint_pos
float64[] joint_vel
float64[] joint_tor

# Camera position
float64[] camera_pos